specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -3047,6 +3096,62 @@
           "format": "int32"
         }
       }
+    },
+    "Stop8A2431BC39114B709A447C12F99E3BB4": {
```

Comment:
Can you please share why the resource definition have this name and not a user friendly name?

Reply:
changed name